README.md

```diff
@@ -2,130 +2,131 @@
 
 # Kata Containers
 
-* [Raising issues](#raising-issues)
-* [Kata Containers repositories](#kata-containers-repositories)
-    * [Code Repositories](#code-repositories)
-        * [Kata Containers-developed components](#kata-containers-developed-components)
-            * [Agent](#agent)
-            * [KSM throttler](#ksm-throttler)
-            * [Runtime](#runtime)
-            * [Trace forwarder](#trace-forwarder)
-        * [Additional](#additional)
-            * [Kernel](#kernel)
-    * [CI](#ci)
-    * [Community](#community)
+* [Kata Containers](#kata-containers)
+    * [Introduction](#introduction)
+    * [Getting started](#getting-started)
     * [Documentation](#documentation)
-    * [Packaging](#packaging)
-    * [Test code](#test-code)
-    * [Utilities](#utilities)
-        * [OS builder](#os-builder)
-    * [Web content](#web-content)
+    * [Community](#community)
+    * [Getting help](#getting-help)
+        * [Raising issues](#raising-issues)
+            * [Current version](#current-version)
+            * [Kata Containers 1.x versions](#kata-containers-1x-versions)
+    * [Developers](#developers)
+        * [Components](#components)
+            * [Kata Containers 1.x components](#kata-containers-1x-components)
+        * [Common repositories](#common-repositories)
 
 ---
 
 Welcome to Kata Containers!
 
-The purpose of this repository is to act as a "top level" site for the project. Specifically it is used:
-
-- To provide a list of the various *other* [Kata Containers repositories](#kata-containers-repositories),
-  along with a brief explanation of their purpose.
-
-- To provide a general area for [Raising Issues](#raising-issues).
+This repository is the home of the Kata Containers code.
```

Comment:
I know this was discussed on earlier version, but somehow GitHub seems to have dropped my comment. From memory, I was suggesting something like:

> This repository is the home of the Kata Containers code for releases after 2.0. Earlier releases used [separate repositories for individual components](#kata-containers-1x-components).

As I wrote in my other (possibly lost) comment, makes me wish we had used git submodules or git subtrees 😄